lib/matplotlib/transforms.py

```diff
@@ -1101,7 +1101,7 @@ def __init__(self, bbox, transform, **kwargs):
         bbox : `Bbox`
         transform : `Transform`
         """
-        if not bbox.is_bbox:
+        if not hasattr(bbox, 'is_bbox') or not bbox.is_bbox:
```

Comment:
You could do this with `getattr`, I think?
```suggestion
        if not getattr(bbox, 'is_bbox', False):
```

Reply:
Ahh, clever! (A bit too tired, so trying to do "simple" things, but they do not always turn out that good...)